server/app/views/v1/stacks/_stack.json.jbuilder

```diff
@@ -15,5 +15,11 @@ json.variables latest_rev.variables
 json.services stack.grid_services.to_a do |grid_service|
   json.partial! 'app/views/v1/grid_services/grid_service', grid_service: grid_service
 end
-json.parent stack.has_parent? ? { name: stack.parent_name } : nil
-json.children stack.children.pluck(:name).map { |child_name| { name: child_name } }
+
+json.parent do
+  json.partial! 'app/views/v1/stacks/parent', stack: stack
```

Comment:
I don't think this particularly needs to be a `partial!`, it's only ever used in one place, so this just makes this harder to read because you have to open up the separate partial files as well now.